MinimedKit/PumpEvents/TempBasalPumpEvent.swift

```diff
@@ -9,10 +9,17 @@
 import Foundation
 
 public class TempBasalPumpEvent: PumpEvent {
+  
+  public enum RateType : String {
+    case Absolute = "absolute"
```

Comment:
Are these raw enumerations part of the Nightscout model? If not, and they're just for debugging, Swift will automatically stringify enum cases when you extend `String`:
```swift
enum RateType: String {
    case Absolute  // = "Absolute"
    case Percent  // = "Percent"
}
```